packages/edit-site/src/components/sidebar-navigation-screen-main/index.js

```diff
@@ -61,7 +61,11 @@ export default function SidebarNavigationScreenMain() {
 						</SidebarNavigationItemGlobalStyles>
 						<NavigatorButton
 							as={ SidebarNavigationItem }
-							path="/page"
+							path={
+								window?.__experimentalAdminViews
+									? '/pages'
+									: '/page'
```

Comment:
Feels like the path should be the same and always `/page` and instead, the rendered component that changes depending on the experiment.

Reply:
Updated the approach to use `/page` as the only path.